common/djangoapps/util/milestones_helpers.py

```diff
@@ -175,6 +175,18 @@ def fulfill_course_milestone(course_key, user):
         milestones_api.add_user_milestone({'id': user.id}, milestone)
 
 
+def remove_course_milestones(course_key, user):
```

Comment:
@saleem-latif if we are going to keep this method we should add a third param `relationship` as we did in `remove_course_content_user_milestones` to allow removal of fullfiled or required course milestone.